src/app/onion/core/learning-object.service.ts

```diff
@@ -183,7 +183,7 @@ export class LearningObjectService {
     query?: string,
     childId?: string
   ): Promise<LearningObject[]> {
-    const route = LEGACY_USER_ROUTES.GET_MY_LEARNING_OBJECTS(authorUsername, filters, query, childId);
+    const route = LEGACY_USER_ROUTES.GET_MY_LEARNING_OBJECTS(authorUsername, filters, query);
```

Comment:
Should be updated to not be a legacy route

Reply:
+1